### PR TITLE
Remove provisioning CIDR from ironicendpoint VIP

### PR DESCRIPTION
**Pull request by kashifest**
This is a bug. kube-proxy using ipvs requires the vip to be /32.  We need to remove the provisioning CIDR from ironicendpoint VIP.

### CODE REVIEW

File: resources/keepalived-docker/manage-keepalived.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-export assignedIP="$PROVISIONING_IP/$PROVISIONING_CIDR"
+export assignedIP="$PROVISIONING_IP"
```
Comment (namnx228) [inline resources/keepalived-docker/manage-keepalived.sh:2]: change to $PROVISIONING_IP/32 I guess

Comment (kashifest) [reply resources/keepalived-docker/manage-keepalived.sh:2]: We need to wait for @maelk on this, he knows the actual bug and his suggestion was to remove it.

Comment (kashifest) [reply resources/keepalived-docker/manage-keepalived.sh:2]: Ah sorry you are right @namnx228

Comment (kashifest) [reply resources/keepalived-docker/manage-keepalived.sh]: Done

**Comment by furkatgofurov7** [comment]
Resizing has failed again. Re-triggering it once again, until patch for resize is in place. 
/test-integration

**Review by maelk — APPROVED** [review]

**Pull request merged**